tests/webview/test_google_redirect.py

```diff
@@ -0,0 +1,18 @@
+from requests import get
+from tests import markers
+
+
+@markers.webview
+@markers.nondestructive
+def test_google_redirect_archive_results(webview_base_url, chrome_options):
+    # end-to-end integration test for redirecting google away from archive URLs
+    # see: https://github.com/openstax/cnx/issues/209
+
+    url = "https://archive-qa.cnx.org/contents/fea3130c-6e57-41b2-a00f-0267ffae273c@5"
+    # TODO: do I need to clear/purge the varnish cache before running this test?
```

Comment:
This shouldn't be necessary. The varnish configuration doesn't cache the redirect response, right?

A configuration change to Varnish is going to restart/reload the service. So the cache would be flushed at that time. Changes to this behavior should be reflected by this test.

Reply:
The TODO comment needs to be removed in that case.

Reply:
Done, removed comment.